tests/frontends/test_pytorch.py

```diff
@@ -418,85 +418,6 @@ def forward(self, x):
         return x
 
 
-def test_module_2_layer_mlp_seq_fwd():
```

Comment:
Please don't delete tests like this.  Mark them as xfail or something like that.

If you really think they have no more value, then please explain why.